azure_rm_aks/aks_create.yml

```diff
@@ -0,0 +1,43 @@
+# Description
+# ===========
+# This playbook create an Azure Kubernetes Service cluster of 2 nodes with provided username and ssh key.
+# Change variables below to customize your AKS deployment.
+# It also requires a valid service principal to create AKS cluster, so fill:
+# - client_id
+# - client_secret
+# This sample requires Ansible 2.6 
+
+- name: Create Azure Kubernetes Service
+  hosts: localhost
+  connection: local
+  vars:
+    resource_group: "{{ resource_group_name }}"
+    location: eastus
+    aks_name: myAKSCluster
+    username: azureuser
+    ssh_key: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDH14vPP1NE+qc3SgpoSPYMVaPLj8ouA1azujnMPbVKtN8mSg0KBWVWebx+GLqGI9nrKwmVwPHVzM8LhbbpYhfnN0UA2OV7sxBIshy7BshDTewiLyR9FXuNjuf8Cwd7WI+vgkKO0Yh6EdgwZ2hz+yVTn+5umCRDOg0ht0i2kyPMn1MHv61aN3noYp5zV8W3Tg8oJ2DPei6FxcNheRWkyvjan73/rISQJYWURlI90b3MCQOTXuI2w6BIxF3kHIHLUtzD7FqQhZdEkM/8cA9wUDgnS6+3ySkGOuQazkEWYn60Ll/597u2fLsq6P/pOCM1+KWLf2kY6mTl/dNxiIdV1d01
+    client_id: "{{ your_client_id }}"
```

Comment:
"{{ lookup('env', 'AZURE_CLIENT_ID') }}"

We need to make sure all the sample can be run on the integration test, so here retrieve the credentials from env.

Reply:
OK, I'll change them.